src/utils/sigma.utils.js

```diff
@@ -591,6 +591,24 @@
   };
 
   /**
+   * The pixel ratio of the screen. Taking zoom into account
```

Comment:
Isn't this possible to cache the pixel ratio once rather than computing it every time? Can the pixel ratio change through time?